### ISSUE TITLE
Rotations: Exclude specific days of week

### ISSUE THREAD

**Issue by traylz** [target/goalert#55]
**Is your feature request related to a problem? Please describe:**
I need a rotation on weekdays only, currently it's not possible to do that

**Describe the solution you'd like:**
When creating rotation I would like to specify working days (e.g Mon-Fri) on which rotation should take place. Days out of rotations should have no shift at all

**Comment by donna-roisen** [comment]
This is actually possible today.  If you look at your rotation and go to the 'edit', you can select the days and hours that the rotation is active.  This view is only available after a rotation has been assigned to a schedule.  So from your schedule, click on the "ASSIGNMENTS", see your rotation hit the ellipsis and select EDIT.  
![2019-07-17_09-10-46](https://user-images.githubusercontent.com/48256056/61383003-75a58d00-a873-11e9-8a73-81b540ce8a2b.png)

**Comment by AdamLWestman** [comment]
@traylz does what Donna explained meet your needs?  

You've obviously highlighted a situation where it would help if we published end user documentation on how to use GoAlert.  We'll work on getting these docs published.

**Comment by traylz** [comment]
Hey! Yeah, checked it. It's almost what I need. Although if I have daily rotation and I exclude weekend from schedule, some people will miss their shifts (e.g. given I have 14 people on rotation - 4 people of 14 will never get a shift), right?

**Comment by donna-roisen** [comment]
yes - that is correct.  Right now rotations are separate from schedules, so it just rotates through at the set frequency (daily, weekly etc.) It is something that we are thinking about but do not currently have prioritized.

**Comment by mburgess00** [comment]
I would love to see this functionality. We would probably consider using this project if it was able to schedule around weekends.

**Comment by pnengchu** [comment]
On the roadmap to merge rotation features into schedules. Once complete this functionality will be covered.
